<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>971</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(77, 77, 127);
</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>811</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(255,255,255,50);
border-radius:50px
</string>
    </property>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mobile Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Email id</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>939</x>
     <y>0</y>
     <width>261</width>
     <height>800</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#23272A;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="createData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:&quot;30px&quot;
</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:&quot;30px&quot;</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteData">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:&quot;30px&quot;</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="homepage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>200</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;
font: 16pt &quot;MS Shell Dlg 2&quot;;
border-radius:&quot;30px&quot;</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
